docs/content/configuration/auth.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: doc_page
+---
+
+# Authentication and Authorization
+
+|Property|Type|Description|Default|Required|
+|--------|-----------|-------|
+|`druid.auth.enabled`|boolean|Determines if authentication and authorization checks will be performed on requests.|false|no|
+|`druid.auth.authenticationChainPath`|String|Path to a file containing a JSON list of Authenticator objects|null|yes|
```

Comment:
Changed this to use a list of JSON Strings, Authenticators are now created through injector and @Named annotations, extension implementers can define their own config classes for their specific implementations.